hetio/neo4j.py

```diff
@@ -287,6 +287,29 @@ def construct_unique_nodes_clause(metarels, unique_nodes):
 
     return unique_nodes_query
 
+def create_path_return_clause(path_style='list'):
+    """
+    Create a Cypher query clause to return paths either as a list or as a string.
+    As formatting the output as a string is less efficient in terms of database
+    hits, 'list' is the default style
+
+    Parameters
+    ----------
+    path_style : str
+        the way the user wants the path returned. Currently supported options
+        are 'list' and 'string'
```

Comment:
Let's add another argument for `node_property`, to allow specifying either `identifier` or `name`. When working with the node lists programatically, we will want to use node identifiers rather than names (names are not gauranteed to be unique and are not ideal for lookup).